tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -433,6 +438,7 @@ def test_conv_ws(
         enable_subblock_padding=False,
         reshard_if_not_optimal=True,
         act_block_w_div=act_block_w_div,
+        act_block_h_override=32,
```

Comment:
why do we have this?

Reply:
To make sure that it is being ignored correctly.

Reply:
can you add comments with this that width sharding ignores this parameter -- we need it documented.

Reply:
Added comment in conv2d.hpp